project13/ViewController.swift

```diff
@@ -4,16 +4,191 @@
 //
 //  Created by Felipe Gil on 2021-07-09.
 //
-
+import CoreImage
 import UIKit
+//TODO
+#warning("Make the Change Filter button change its title to show the name of the currently filter")
+#warning("Try more than 1 slider with different properties")
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var intensity: UISlider!
```

Comment:
I have not noticed the slider was named `intensity`. This is not very descriptive. Please refer to the naming links below.